astropy/table/tests/test_row.py

```diff
@@ -279,3 +279,29 @@ def test_row_tuple_column_slice_transaction():
         t[1]['a', 'b'] = (-1, -1 * u.s)  # Bad unit
     assert "'s' (time) and 'm' (length) are not convertible" in str(err)
     assert t[1] == tc[1]
+
+def test_uint_indexing():
+    """
+    Test that accessing a row with an unsigned integer
+    works as with a signed integer.  Similarly tests
+    that printing such a row works.
+
+    This is non-trivial: adding a signed and unsigned
+    integer in numpy results in a float, which is an
+    invalid slice index.
+    """
+    t = table.Table([[1.,2.,3.]], names='a')
```

Comment:
The most total nitpick ever, but spaces after `,`

More relevantly: add to the docstring a backreference to the github issue, something like
```
Regression test for gh-7464.
```